ui/app/AppLayouts/Chat/controls/community/HoldingsDropdown.qml

```diff
@@ -0,0 +1,263 @@
+                // Tokens layout definition:
+                ColumnLayout {
+                    Layout.fillWidth: true
+                    Layout.fillHeight: true
+                    StatusPickerButton {
+                        Layout.fillWidth: true
+                        Layout.topMargin: 8
+                        Layout.preferredHeight: 36
+                        bgColor: Theme.palette.baseColor5
+                        contentColor: Theme.palette.directColor1
+                        text: root.tokenName
+                        textPixelSize: 13
+                        image.source: root.tokenImage
+                        onClicked: loader.sourceComponent = tokensExtendedView
+                    }
+
+                    // TODO: Update preferredHeight according to design once `StatusInput` behaves correclty with different heights.
+                    StatusInput {
+                        Layout.fillWidth: true
+                        //Layout.preferredHeight: 36
+                        text: root.tokenAmountValue
+                        font.pixelSize: 13
+                        rightPadding: amountText.implicitWidth + amountText.anchors.rightMargin + leftPadding
+                        input.placeholderText: "0"
+                        validationMode: StatusInput.ValidationMode.IgnoreInvalidInput
+                        validators: StatusFloatValidator {  bottom: 0 }
+
+                        StatusBaseText {
+                            id: amountText
+                            anchors.right: parent.right
+                            anchors.rightMargin: 13
+                            anchors.verticalCenter: parent.verticalCenter
+                            text: qsTr("Amount")
+                            color: Theme.palette.baseColor1
+                            font.pixelSize: 13
+                        }
+                        onTextChanged: root.tokenAmountValue = text
+                    }
+                    // Just a filler
+                    Item { Layout.fillHeight: true}
+                    // TODO: Needed `StatusButton` redesign that allows to fill the width.
+                    StatusButton {
+                        enabled: root.ready
+                        text: qsTr("Add")
+                        height: 44
+                        Layout.alignment: Qt.AlignHCenter
+                        //Layout.fillWidth: true
+                        onClicked: { root.addToken(root.tokenAmountValue + " " + root.tokenName, root.tokenImage, root.operator) }
+                    }
+                } // End of Tokens Layout definition
+
+                // TODO
+                Rectangle {
+                    Layout.fillWidth: true
+                    Layout.fillHeight: true
+                }
+
+                // TODO
+                Rectangle {
+                    Layout.fillWidth: true
+                    Layout.fillHeight: true
+                }
```

Comment:
Well, since this task is part of a bigger one, yes.. These are the next steps of the task (we are integrating this task to a feature one where I will be working on them).